### PR TITLE
Modify agent initialization file for v2

### LINKED ISSUE #70: Modify agent initialization file for v2
Remove unused options from f5-openstack-agent.ini file.

### PR DESCRIPTION
**Pull request by jlongstaf**
@richbrowne @mattgreene @jputrino 
#### What issues does this address?
Fixes #70

#### What's this change do?
Removes unused agent ini options.

#### Any background context?
LBaaS v1 code allowed using iApps and BIG-IQ for configuring BIG-IPs, but
LBaaS v2 does not.

Issues:
Fixes #70

Problem: Agent ini file contains options that are not used in LBaaS v2.

Analysis: Removed unused options (BIG-IQ, etc.). Fixed some spelling
errors.

Tests:

### CODE REVIEW

**Comment by mattgreene** [comment]
Looks fine to me, but I'll let Rich review and merge the commit.

File: etc/neutron/services/f5/f5-openstack-agent.ini [hunk 1]
```diff
@@ -332,8 +324,7 @@ f5_external_physical_mappings = default:1.1:True
 #
 # Device Tunneling (VTEP) selfips
 #
-# This is a single entry or comma separated list of cidr (h/m) format
-# selfip addresses, one per BIG-IP® device, to use for VTEP addresses.
+# This is the name of a selfip address to use for VTEP addresses.
```
Comment (jputrino) [inline etc/neutron/services/f5/f5-openstack-agent.ini]: Does this need to be 'a BIG-IP® selfip address`?

File: etc/neutron/services/f5/f5-openstack-agent.ini [hunk 2]
```diff
@@ -379,8 +370,8 @@ f5_vtep_selfip_name = 'vtep'
 #
 # This is a boolean entry which determines if they BIG-IP® will use
```
Comment (jputrino) [inline etc/neutron/services/f5/f5-openstack-agent.ini]: 'the BIG-IP', not 'they BIG-IP'

File: etc/neutron/services/f5/f5-openstack-agent.ini [hunk 3]
```diff
@@ -379,8 +370,8 @@ f5_vtep_selfip_name = 'vtep'
...
 # L2 Population service to update its fdb tunnel entries. This needs
-# to be setup in accordance with the way the other tunnel agents are
-# setup.  If the BIG-IP® agent and other tunnel agents don't match
+# to be set up in accordance with the way the other tunnel agents are
+# set up.  If the BIG-IP® agent and other tunnel agents don't match
 # the tunnel setup will not work properly.
 #
 l2_population = True
```
Comment (jputrino) [inline etc/neutron/services/f5/f5-openstack-agent.ini]: Can we change 'Global Routing Mode' to 'Global Routed Mode' so it matches the actual setting?

File: etc/neutron/services/f5/f5-openstack-agent.ini [hunk 4]
```diff
@@ -486,10 +477,10 @@ f5_route_domain_strictness = False
 #
```
Comment (jputrino) [inline etc/neutron/services/f5/f5-openstack-agent.ini]: line 468: 'or else', not 'or less'

File: etc/neutron/services/f5/f5-openstack-agent.ini [hunk 5]
```diff
@@ -486,10 +477,10 @@ f5_route_domain_strictness = False
...
 # If this is set to False, a SNAT will not
 # be created (routed mode) and the BigIP
```
Comment (jputrino) [inline etc/neutron/services/f5/f5-openstack-agent.ini]: BIG-IP®

File: etc/neutron/services/f5/f5-openstack-agent.ini [hunk 6]
```diff
@@ -486,10 +477,10 @@ f5_route_domain_strictness = False
...
-# will attempt to setup a floating SelfIP
+# will attempt to set up a floating SelfIP
 # as the subnet's default gateway address.
 # and a wild card IP forwarding virtual
-# server will be setup on member's network. 
+# server will be set up on member's network.
 # Setting this to False will mean Neutron
 # Floating Self IPs will not longer work 
 # if the same BigIP device is not being used
```
Comment (jputrino) [inline etc/neutron/services/f5/f5-openstack-agent.ini]: BIG-IP®

File: etc/neutron/services/f5/f5-openstack-agent.ini [hunk 7]
```diff
@@ -609,7 +600,7 @@ f5_bigip_lbaas_device_driver = f5_openstack_agent.lbaasv2.drivers.bigip.icontrol
 # If a single IP address is used and the HA model 
 # is not standalone, all devices in the sync failover
 # device group for the hostname specified must have 
-# there management IP address reachable to the agent.
+# their management IP address reachable to the agent.
 # If order to access devices' iControl® interfaces via
 # SelfIPs, you should specify them as a comma
```
Comment (jputrino) [inline etc/neutron/services/f5/f5-openstack-agent.ini]: We need to make sure we're referencing selfips consistently throughout. So far, I've seen 'selfip', 'SelfIP' and 'Self IP'.

**Comment by jputrino** [comment]
I have a few comments; once those are addressed this should be good to go (from a doc standpoint).

**Pull request merged**